INSTALL.md

```diff
@@ -413,16 +413,34 @@ For a more detailed guide to configuring your server for federation, see
 
 ## Email
 
-It is desirable for Synapse to have the capability to send email. For example,
-this is required to support the 'password reset' feature.
+It is desirable for Synapse to have the capability to send email. This allows
+Synapse to send password reset emails, register users via their email address,
+and send email notifications to users when they receive new messages. 
+
+Email can be set up in two ways, either by giving Synapse SMTP details
+(recommended), or delegating another server to send emails on your behalf. Note
+that the latter will be removed in a future Synapse version.
 
 To configure an SMTP server for Synapse, modify the configuration section
-headed ``email``, and be sure to have at least the ``smtp_host``, ``smtp_port``
-and ``notif_from`` fields filled out. You may also need to set ``smtp_user``,
-``smtp_pass``, and ``require_transport_security``.
+headed `email`, and be sure to have at least the `smtp_host`, `smtp_port`
+and `notif_from` fields filled out.  You may also need to set `smtp_user`,
+`smtp_pass`, and `require_transport_security`.
+
+To delegate an external server to handle email sending, modify the
```

Comment:
frankly I'd get rid of this, given it's deprecated anyway.

Reply:
Yeah, probably more relevant to existing installs instead of new ones.